studio/util.py

```diff
@@ -521,3 +523,31 @@ def rm_rf(path):
         shutil.rmtree(path)  # remove dir and all contains
     else:
         raise ValueError("file {} is not a file or dir.".format(path))
+
+class LogReprinter(object):
+    def __init__(self, log_path):
```

Comment:
Factor out log reprinter logic into separate util class.
Will be used from local_worker module.